sdk/core/System.ClientModel/samples/Configuration.md

```diff
@@ -0,0 +1,89 @@
+# System.ClientModel-based client configuration samples
+
+## Configuring retries
+
+To modify the retry policy, create a new instance of `ClientRetryPolicy` and set it on the `ClientPipelineOptions` passed to the client constructor.
+
+By default, clients will retry a request three times using an exponential retry strategy with an initial delay of 0.8 seconds and a maximum delay of one minute.
```

Comment:
Should we mention jitter here also?

Reply:
ClientModel doesn't use jitter today: https://github.com/Azure/azure-sdk-for-net/issues/41393